lib/components/providers/theme/themeProvider.tsx

```diff
@@ -0,0 +1,31 @@
+import { type ReactNode, useEffect, useState } from "react";
+
+import { ThemeContext, initialThemeState } from "./themeContext";
+
+type Props = {
+    children: ReactNode;
+};
+
+export const ThemeProvider = ({ children }: Props) => {
```

Comment:
```suggestion
export const ThemeProvider = ({ children }: PropsWithChildren) => {
```
Don't forget to import from React.